api.planx.uk/modules/analytics/metabase/collection/service.ts

```diff
@@ -0,0 +1,36 @@
+import { createMetabaseClient } from "../shared/client.js";
+import { updateMetabaseId } from "./updateMetabaseId.js";
+import type { NewCollectionParams } from "./types.js";
+import { getTeamIdAndMetabaseId } from "./getTeamIdAndMetabaseId.js";
+import { createCollection } from "./createCollection.js";
+
+const client = createMetabaseClient();
```

Comment:
I don't think this line is needed, the `client` doesn't appear to be used anywhere

Reply:
A better pattern in all these places where client is used - see the `getClient` pattern in this file: https://github.com/theopensystemslab/planx-new/blob/47aba73d1eefdf8782b12493264a925e1a58969c/api.planx.uk/modules/flows/moveFlow/service.ts#L7